src/stores/config.ts

```diff
@@ -15,6 +15,7 @@ import { i18n } from '@/i18n';
         collection: '',
         manifest: '',
         item: '',
+        showAnnotationIcons: false,
```

Comment:
I think this is not what was requested. In #560 @mweidling is suggesting to omit the "icon" key on each annotation type config. TIDO then automatically recognizes that no icon should be rendered there. This enhances the modular nature of the config since you can decide per panel what to display.